config.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+
+class Config(object):
+    NAME = os.getenv('NAME', "rm-comms-template")
+    VERSION = os.getenv("VERSION", "0.0.1")
+    SCHEME = os.getenv("SCHEME", "http")
+    HOST = os.getenv("HOST", "0.0.0.0")
+    PORT = os.getenv("PORT", 8081)
+    DEBUG = os.getenv("DEBUG", False)
+    SCHEMA = os.getenv("SCHEMA", "templatesvc")
+    SQLALCHEMY_DATABASE_URI = os.getenv("SQLALCHEMY_DATABASE_URI",
+                                        "postgres://postgres:postgres@postgres:5432/postgres")
+    SQLALCHEMY_TRACK_MODIFICATIONS = os.getenv("SQLALCHEMY_TRACK_MODIFICATIONS", False)
```

Comment:
Does this need to be an environmental variable, do we think it will ever change?

Reply:
Recommended to explicitly put in by flask-sqlalchemy, its going to be deprecated but was causing noise in the logs without it.

Reply:
Sorry didn't actually twig that you said about the environment variable, good point.